versions/latest.json

```diff
@@ -0,0 +1,26 @@
+{
+  "subnet-evm": "v0.7.3",
```

Comment:
as said, I believe the file should be:
- latest_rpc
- map from rpc -> network -> versions for all apps

Reply:
Initially I chose to have all networks options for subnetevm, but found that in our codebase, we only use latest version of subnet evm in https://github.com/ava-labs/avalanche-cli/blob/main/pkg/vm/evm_prompts.go#L887, which is called during blockchain create command. At this stage, user has not chosen what network they want to deploy the blockchain in, and therefore it doesn't make sense for us to have different versions of subnet evm used for different networks